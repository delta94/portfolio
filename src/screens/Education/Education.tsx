import React, { useContext } from 'react';
import { IoMdSchool } from 'react-icons/io';
import { InformationsContext } from '../../App';
import { InformationBarContainer } from '../../components';
import { theme } from '../../theme';
import { Informations } from '../../utils/informations';

interface EducationProps {}

const EducationIcon: () => JSX.Element = () => <IoMdSchool size={30} color={theme.colors.thirdColor} />;

const Education: React.FC<EducationProps> = () => {
  const context: Informations | undefined = useContext<Informations | undefined>(InformationsContext);
  return (
    <>
      {context?.educationInformations && (
        <InformationBarContainer
          icon={<EducationIcon />}
          marginTop={6}
          title='Education'
          informations={context.educationInformations}
        />
      )}
    </>
  );
};

export { Education };
