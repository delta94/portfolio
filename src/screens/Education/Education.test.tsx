import { render } from '@testing-library/react';
import React from 'react';
import { TestContextProvider } from '../../TestContextProvider';
import { Education } from './Education';

test('Education renders correctly', () => {
  const { asFragment } = render(
    <TestContextProvider>
      <Education />
    </TestContextProvider>,
  );
  expect(asFragment()).toMatchSnapshot();
});
