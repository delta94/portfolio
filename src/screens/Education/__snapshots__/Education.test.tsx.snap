// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education renders correctly 1`] = `<DocumentFragment />`;
